integrations/pom.xml

```diff
@@ -51,6 +51,7 @@
         <module>micrometer</module>
         <module>vault</module>
         <module>microstream</module>
+        <module>kotlin-support</module>
```

Comment:
having a single module may be strange (not sure how dependencies work in Kotlin, but we probably do not want to add the whole Helidon world to any kotling app).
We should at least separate SE and Níma.

Reply:
Introduced three different modules.